pom.xml

```diff
@@ -15,6 +15,12 @@
   ~ limitations under the License.
   --><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
```

Comment:
I would move the dependencies listed in this POM to `vertx-junit5` (and the ones needed in the other module if need be).

You may also need `dependencyManagement` if you have shared dependencies and you want to factor dependencies.

Reply:
I've moved in parent pom only dependencies shared between modules, this is what you mean?